lib/result.js

```diff
@@ -70,8 +70,10 @@ class Result {
     return this;
   }
 
-  _finalizeMetadata(instance) {
-    this.url = instance.getURL();
+  _finalizeMetadata(instance) {
+    if (this.booted) {

```

Comment:
I went with this approach versus using `instance._booted` since it's marked private